myproject/home.php

```diff
@@ -69,6 +69,7 @@
 echo "Процент от общего числа билетов:".$procent."%<br>";
 */
 echo "Массивы<br>";
+echo "1.<br>";
 $n=9;
 $mas = array();
 for ($i=0; $i<=$n; $i++)
```

Comment:
Code style. You sometimes use spaces and sometimes don't use. It would be better to use one style (with spaces).
`$mas = array();` - it's ok
But
`$n=9;` -> `$n = 9;`
`for ($i=0; $i<=$n; $i++)` -> `for ($i = 0; $i <= $n; $i++)`